core/webhooks/webhooks.go

```diff
@@ -43,12 +28,14 @@ type WebhookChatMessage struct {
 }
 
 func SendEventToWebhooks(payload WebhookEvent) {
-	for _, webhook := range config.Config.Webhooks {
+	webhooks := data.GetWebhooksForEvent(payload.Type)
+
+	for _, webhook := range webhooks {
 		log.Debugf("Checking Webhook %s to send event: %s", webhook.Url, payload.Type)
 
-		eventsAccepted := strings.Join(webhook.Events, ",")
+		eventsAccepted := strings.Join(interface{}(webhook.Events).([]string), ",")
```

Comment:
Is this and the if below even needed any more with the new query you added?

Reply:
No longer needed, thanks for catching!